.NET/Microsoft.Recognizers.Text.Number/Hindi/Extractors/CardinalExtractor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Hindi;
+
+namespace Microsoft.Recognizers.Text.Number.Hindi
+{
+    public class CardinalExtractor : BaseNumberExtractor
+    {
+        private static readonly ConcurrentDictionary<string, CardinalExtractor> Instances =
+            new ConcurrentDictionary<string, CardinalExtractor>();
+
+        private CardinalExtractor(string placeholder = NumbersDefinitions.PlaceHolderDefault)
+        {
+            var builder = ImmutableDictionary.CreateBuilder<Regex, TypeTag>();
+
+            // Add Integer Regexes
+            var intExtract = IntegerExtractor.GetInstance(placeholder);
+            builder.AddRange(intExtract.Regexes);
+
+            // Add Double Regexes
+            // var douExtract = DoubleExtractor.GetInstance(placeholder);
```

Comment:
Is there a particular reason this is commented out?

Reply:
no (commented during development and forgotten), corrected